package.json

```diff
@@ -3,25 +3,24 @@
   "version": "0.0.5",
   "description": "Hapi.js plugin for authentication scheme of accepting connections only from certain IPs",
   "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
   "repository": "https://github.com/chamini2/hapi-auth-ip-whitelist",
   "author": "Matteo Ferrando <matteo.ferrando2@gmail.com>",
   "license": "MIT",
-  "scripts": {
-    "build": "tsc",
-    "prepublish": "npm run build"
-  },
   "devDependencies": {
-    "@types/boom": "^4.3.7",
-    "@types/hapi": "^16.1.12",
-    "@types/ip": "^0.0.30",
-    "typescript": "^2.6.2"
```

Comment:
Why did you remove typescript?

Reply:
TypeScript does not add any value over ES6 in such a small project. It adds dependencies, compilation time and file access while ES6 is used as is by Node.

Reply:
It does not add dependencies, as it is only for development. I do see your point in this being a small project.

Reply:
Dev dependencies are also a form of dependency ^_^
I believe that it clutters the dev process unnecessarily for a project of this scope.
So should I bring back TypeScript?